packages/dashboard/src/components/Restore/RestorePersonaFromLocal.tsx

```diff
@@ -41,12 +38,16 @@ const useStyles = makeStyles()((theme) => ({
         marginTop: 7,
     },
 }))
-export const RestoreFromLocal = memo(function RestoreFromLocal() {
+interface RestoreFromLocalProps {
+    handleRestoreFromLocalStore: () => Promise<void>
```

Comment:
This component is `RestorePersonaFromLocal`, you can omit the prefix
```suggestion
    onRestore: () => Promise<void>
```